build.sbt

```diff
@@ -107,6 +107,7 @@ lazy val core = myCrossProject("core")
 lazy val plugin = myCrossProject("plugin")
   .settings(noPublishSettings)
   .settings(
+    addSbtPlugin(Dependencies.scalafixPlugin),
```

Comment:
Adding sbt-scalafix to the build here does not mean that its settings or tasks can be used in `StewardPlugin.scala`. The reason is that this file is added as resource to the `core` project and while Scala Steward is running this source file is added as global project to `$HOME/.sbt/1.0/plugins/`. As far as I know, these global source-based plugins can't declare dependencies on other plugins it can only use settings and keys of ther global plugins (that was the reason why it could use sb-updates' settings before, because it was also installed as global plugin).

It would be great if we could support `scalafixDependencies` without adding sbt-scalafix as global plugin for all projects. The reason I'm wary of adding `sbt-scalafix` as global plugin is that I've seen it breaking projects that already used that plugin locally. I'm wondering if sbt settings/tasks can be called by name as `String` if they are defined in the build. That way we could get `scalafixDependencies` only in projects that already depend on sbt-scalafix

Reply:
https://github.com/fthomas/scala-steward/pull/602 documents why I'd like to avoid adding sbt-scalafix as global plugin.

Reply:
This is a very good remark. I overlooked that problem.

I will find a way to not install `sbt-scalafix` as a global plugin.

Reply:
was more complicated, that I first thought.